README.md

```diff
@@ -38,3 +48,26 @@ To generate a file for test purpose, especially with a fixed file size or random
 To download file from multiple storage nodes **in parallel**, `--node` option supports to specify multiple comma separated URLs, e.g. `url1,url2,url3`.
 
 If you want to verify the **merkle proof** of downloaded segment, please specify `--proof` option.
+
+**Write to KV**
+
+By storage node urls:
+
+```
+./0g-storage-client kv-write --url <blockchain_rpc_endpoint> --contract <0g-storage_contract_address> --key <private_key> --node <storage_node_rpc_endpoint> --stream-id <stream_id> --stream-keys <stream_keys> --stream-values <stream_values>
+```
+
+By indexer:
+```
+./0g-storage-client kv-write --url <blockchain_rpc_endpoint> --contract <0g-storage_contract_address> --key <private_key> --indexer <indexer_rpc_endpoint> --stream-id <stream_id> --stream-keys <stream_keys> --stream-values <stream_values>
```

Comment:
Why writing to storage kv should have a storage node endpoint? Shouldn't kv have a config of storage nodes that it can connect to ?

Reply:
When writing to kv stream, the kv operations are serialized into a local data piece and then uploaded to the storage as a normal file.